curriculum/challenges/english/25-front-end-development/lab-real-time-counter/66bb3e20d3dc5b6d0a21f5dd.md

```diff
@@ -43,7 +43,8 @@ assert.equal(charCount.id, 'char-count');
 The `#char-count` `p` element should initially contain the text `Character Count: 0/50`.
 
 ```js
-assert(document.querySelector('p')?.innerText, 'Character Count: 0/50');
+const charCount = document.getElementById('char-count');
+assert.equal(charCount, 'Character Count: 0/50');
```

Comment:
an html element is never going to eb equal to a string, there was nothing to delete to add the assert